package.json

```diff
@@ -15,11 +15,11 @@
   "scripts": {
     "test": "./node_modules/.bin/elm-test tests/RunTestsInTerminal.elm",
     "start": "node server.js",
-    "client": "$(npm bin)/webpack-dev-server --hot --inline",
-    "server": "$(npm bin)/nodemon server.js",
-    "dev": "$(npm bin)/nf start",
-    "build": "$(npm bin)/webpack --progress --display-modules",
-    "postinstall": "$(npm bin)/elm-package install -y && yarn build"
+    "client": "webpack-dev-server --hot --inline",
```

Comment:
maybe fix this on master too ?
As I was saying there is a bug on this branch with the autocomplete menu that I haven't been able to fix.
I have already made most of the changes in this branch on master. I haven't made the changes that isolate autocompleteLang and route. I think I will make a branch and make the changes according to the rfeldman architecture